tests/unit/CRR/ReplicationStatusUpdater.js

```diff
@@ -83,24 +88,38 @@ describe('ReplicationStatusUpdater', () => {
     });
 
     it('should process bucket for CRR with multiple objects', done => {
-        crr.s3.listObjectVersions = jest.fn((params, cb) => cb(null, listVersionsRes));
+        crr = initializeCrrWithMocks({
+            buckets: ['bucket0'],
+            workers: 10,
+            replicationStatusToProcess: ['NEW'],
+            targetPrefix: 'toto',
+            listingLimit: 10,
+            siteName: 'aws-location',
+        }, logger, {
+            ListObjectVersionsCommand: listVersionsRes, // Use custom response for list object versions
```

Comment:
```suggestion
            ListObjectVersionsCommand: listVersionsRes,
```

That's the standard of the function and the signature of the function